# ABB IRB6660 205
robot_manipulator:
  kinematics_solver: moveit_opw_kinematics_plugin/MoveItOPWKinematicsPlugin
  opw_kinematics_geometric_parameters:
    a1:  0.300
    a2: -0.280
    b:   0.000
    c1:  0.8145
    c2:  0.700
    c3:  0.893
    c4:  0.200
  opw_kinematics_joint_offsets: [0.0, 0.0, -1.57079632679, 0.0, 0.0, 0.0]
  opw_kinematics_joint_sign_corrections: [1, 1, 1, 1, 1, 1]

robot_end_effector:
  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005

# robot_manipulator:
#   kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
#   kinematics_solver_search_resolution: 0.005
#   kinematics_solver_timeout: 0.005